crates/sui-core/src/authority.rs

```diff
@@ -588,6 +595,24 @@ impl AuthorityState {
         // commit_certificate finished, the tx is fully committed to the store.
         tx_guard.commit_tx();
 
+        // Update metrics.
+        self.metrics.total_effects.inc();
```

Comment:
I think this is moved too early btw.  At this point execution hasn't happened yet, it could error out and an effect would not be produced.